### ISSUE TITLE
Inserire il codice in LaTeX

### ISSUE THREAD

**Issue by vincepandolfo** [olimpiadi-informatica/ois#5]
Ciao,
per inserire il codice nella mia soluzione ho usato i tag \begin{lstlisting}[language=C++] e \ending{lstlisting}, solo che la resa grafica di tale soluzione non mi convince molto, anche perché preferirei inserire il codice come è stato fatto nel booklet delle OII 2014. Come si fa? 
Credo anche che sia utile definire un metodo standard per l'inserimento del codice, così lo mettiamo tutti allo stesso modo

**Comment by gabrfarina** [comment]
Lo farò io più avanti (ho scritto un tool apposta).

**Comment by vincepandolfo** [comment]
Perfetto, allora lascio com'è :)

**Comment by gabrfarina** [comment]
Come ti scrivevo nell'[altra issue](https://github.com/olimpiadi-informatica/ois/issues/3), il codice non rispetta le linee guida. D'altra parte ti invio il risultato che ottengo con il mio (nuovo) tool che esporta codice colorato, così possiamo vedere se sta bene (o ad esempio i caratteri siano troppo grandi) :smile: 

(Te lo mando in privato)

**Comment by cip999** [comment]
Potrei avere anch'io un feedback sul risultato del mio codice con il tool? In particolare, mi interesserebbe vedere quanto spazio occupa (possibilmente meno di quello attuale), perché altrimenti dovrei cercare di essere un po' più conciso...  :sweat_smile:

**Comment by gabrfarina** [comment]
@cip999: Certo, volentieri (così testiamo un po' il tool). Io e @wil93 stiamo pensando di farne un'app flask veloce protetta in modo che possiate essere autonomi.
Github non mi lascia allegare pdf, ti scrivo su facebook così ti spiego anche come includere il codice! :wink:

**Comment by gabrfarina** [comment]
@wil93  ed io abbiamo caricato il beautifier sul server, potete trovarlo [QUI](http://cms.di.unipi.it/code2latex/).

**Comment by arn4** [comment]
Quando produce il file dà il nome: "output.pdf.html", quindi di fatto il computer lo interpreta come file html.
Non è nulla di importante perché basta rinominare il file per risolvere, ma volevo sapere se succede solo a me.

**Comment by wil93** [comment]
@arna1998 da me (da chrome) lo salva come `output.pdf`

**Comment by Carlovan** [comment]
Si anche a me succede, anche se non ci aggiunge l'estensione. Forse perchè sono su Linux che delle estensioni se ne frega abbastanza. In ogni caso quando lo salvo mi dice **Tipo file:** _html_

**Comment by wil93** [comment]
Ho fatto una modifica, adesso succede ancora?

**Comment by arn4** [comment]
Ora funziona correttamente! Grazie
